mule-apikit-tools/mule-apikit-scaffolder/src/main/java/org/mule/tools/apikit/Scaffolder.java

```diff
@@ -211,10 +211,10 @@ public void run() {
       muleConfigGenerator.generate();
       muleArtifactJsonGenerator.generate();
     } catch (Exception e) {
```

Comment:
MuleArtifactJsonGenerator and MuleConfigGenerator catch a lot of errors and are being used to just log them. Review those classes.